packages/block-editor/src/components/block-settings/container.native.js

```diff
@@ -30,23 +61,27 @@ function BottomSheetSettings( {
 			contentStyle={ styles.content }
 			{ ...props }
 		>
-			<BottomSheetConsumer>
-				{ ( { currentScreen, extraProps, ...bottomSheetProps } ) => {
-					switch ( currentScreen ) {
-						case colorsUtils.subsheets.color:
-							return (
-								<ColorSettings
-									defaultSettings={ settings }
-									{ ...bottomSheetProps }
-									{ ...extraProps }
-								/>
-							);
-						case colorsUtils.subsheets.settings:
-						default:
-							return <InspectorControls.Slot />;
-					}
-				} }
-			</BottomSheetConsumer>
+			<BottomSheet.NavigationContainer animate>
+				<NavigationContainer theme={ MyTheme }>
+					<Stack.Navigator
+						screenOptions={ {
+							headerShown: false,
+							gestureEnabled: false,
+						} }
```

Comment:
Can we extract that piece of code? The same options are also used in colors-settings.